packages/odyssey-react/src/components/Box/_box-width-height.scss

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.width1px {
+  width: 1px;
+}
+
+.width1em {
+  width: 1em;
+}
+
+.widthFull {
+  width: 100%;
+}
+
+.widthScreen {
+  width: 100vw;
+}
+
+.widthMaxLineLength {
```

Comment:
Do we use this internally? We may want to drop it, as it insists on a fixed width that's rather large.

Instead, users should use `display: block;` with `.maxWidthMaxLineLength`.

Reply:
We don't, dropping